fmriprep/workflows/anatomical.py

```diff
@@ -143,7 +143,7 @@ def t1w_preprocessing(name='t1w_preprocessing', settings=None):
                              interpolation='NearestNeighbor'),
         name='T1_2_MNI_warp'
     )
-    seg_2_mni.inputs.reference_image = op.join(get_mni_template(),
+    seg_2_mni.inputs.reference_image = op.join(get_mni_icbm152_nlin_asym_09c(),
                                                'MNI152_T1_1mm.nii.gz')
```

Comment:
Yeah thats what the problem ended up being. Wasn't careful and didn't change them across the board.